source/administration/config-settings.rst

```diff
@@ -3732,6 +3740,19 @@ Changes made when hardened mode is enabled:
 | This feature’s ``config.json`` setting is ``"ExperimentalEnableHardenedMode": false`` with options ``true`` and ``false`` for above settings respectively.          |
 +---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Limit Access to Client Config Settings Prior to Login
```

Comment:
Maybe drop `Client` from the title? We only really refer to it as the client config internally